README.md

```diff
@@ -57,8 +57,8 @@ There are two different ways of building libpqxx from the command line:
 HP-UX, Irix, Solaris, etc.  Even on Microsoft Windows, a Unix-like environment
 such as WSL, Cygwin, or MinGW should work.
 
-You'll find detailed build and install instructions in `BUILDING-configure.md`
-and `BUILDING-cmake.md`, respectively.
+You'll find detailed build and install instructions in [`BUILDING-configure.md`](/BUILDING-configure.md)
```

Comment:
That's a nice touch.  I never gave much thought about linking between the markdown files because _usually_ the route to the target document depends on the setting in which you access the source document.  But here, it makes absolute sense.

Is the `/` appropriate in this particular path?  Or could we just say `(BUILDING-configure.md)`?

Reply:
I've been using these sorts of internal links for a while. If you do `(BUILDING-configure.md)` they don't really work most places. Most reliable is to always use an explicitly relative link: `(./BUILDING-configure.md)` as that will get github to do the right thing, and wherever else the markdown is processed will also do the right thing

Reply:
I did the corrections as @KayEss suggested.

Reply:
Thanks.